src/cascadia/TerminalSettingsModel/ActionArgs.h

```diff
@@ -594,7 +592,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
         NewTabArgs() = default;
         NewTabArgs(const Model::INewContentArgs& terminalArgs) :
             _ContentArgs{ terminalArgs } {};
-        WINRT_PROPERTY(Model::INewContentArgs, ContentArgs, nullptr);
+        WINRT_PROPERTY(Model::INewContentArgs, ContentArgs, Model::NewTerminalArgs{});

```

Comment:
this ensures that every NewTab action gets a new empty NewTerminalArgs when it is created. Is that correct?

Reply:
The issue is that the settings editor side currently cannot handle an `INewContentArgs` that is not a `NewTerminalArgs` (todo marked for it [here](https://github.com/microsoft/terminal/issues/19056)). I've reverted this back to `nullptr` for now and moved the `NewTerminalArgs` initialization into `GetEmptyArgsForAction`, which only the SUI uses for now